pkg/apis/eventing/v1beta3/eventtype_defaults_test.go

```diff
@@ -43,42 +43,86 @@ func TestEventTypeDefaults(t *testing.T) {
 		"broker empty": {
 			initial: EventType{
 				Spec: EventTypeSpec{
-					Type:   "test-type",
-					Source: testSource,
 					Reference: &duckv1.KReference{
 						APIVersion: "eventing.knative.dev/v1",
 						Kind:       "Broker",
 						Name:       "default",
 					},
-					Schema: testSchema,
+					Attributes: []EventAttributeDefinition{
+						EventAttributeDefinition{
+							Value:    "test-type",
```

Comment:
would `name` not also really make sense here?

should we require name? for the attr def?